gvsbuild/projects/libsoup.py

```diff
@@ -61,8 +61,8 @@ def __init__(self):
         Project.__init__(
             self,
             "libsoup3",
-            archive_url="https://download.gnome.org/sources/libsoup/3.0/libsoup-3.0.6.tar.xz",
-            hash="b45d59f840b9acf9bb45fd45854e3ef672f57e3ab957401c3ad8d7502ac23da6",
+            archive_url="http://ftp.acc.umu.se/pub/GNOME/sources/libsoup//3.2/libsoup-3.2.0.tar.xz",
```

Comment:
Why did we switch to the Umea University mirror instead of using the mirror distribution service?

Reply:
cut&paste from the wrong place :)